### DISCUSSION TITLE
Github Team Account in an organization only for 1 repo

### DISCUSSION THREAD

**Discussion by rizkashifs** [community/community#72642] [New to GitHub]
Dear All,

I have an organization which has 2 repos. I want to enforce branch protection for 1 of these repos (to protect the main branch).  It seems like the users across both repos are getting calculated for the bill. 

Please suggest how I can streamline to this being build for user only for that 1 repo.

Thanks!

**Comment by queenofcorgis** [comment]
Hi 👋🏼 

Where have you set up these rules at the repo level? Or org level?
